charts/sscs-tya-notif/values.yaml

```diff
@@ -49,7 +49,9 @@ java:
     IDAM_API_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
-    JOB_SCHEDULER_DB_CONNECTION_OPTIONS: ""
+    COH_URL: "http://coh-cor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    ONLINE_HEARING_LINK: "https://sscs-cor-frontend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    JOB_SCHEDULER_DB_CONNECTION_OPTIONS: ?sslmode=require
     MAX_ACTIVE_DB_CONNECTIONS: 70
     HOURS_START_TIME: 0
     HOURS_END_TIME: 23
```

Comment:
This is an issue, start time should be 9am end time should be 5pm, on preview it's ok to have the values here